content/applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst

```diff
@@ -0,0 +1,141 @@
+   (incoming) will be titled "San Francisco 2".
+
+Configure products for inter-warehouse replenishment
+====================================================
+
+Products must also be configured properly in order to be transferred between warehouses. Starting
+from the :menuselection:`Inventory` module, go to :menuselection:`Products --> Products` and select
+an existing product or configure a new one by clicking :guilabel:`Create`. On the product form, go
+to the :guilabel:`Inventory` tab and enable the checkbox for :guilabel:`X: Supply Product from Y`,
+with X being the warehouse receiving the transferred products and Y being the warehouse that
+products are transferred from.
+
+.. image:: warehouse_replenishment_transfer/product-transfer-configuration.png
+   :align: center
+   :alt: Enable the checkbox to resupply one warehouse from another.
+
+Replenish one warehouse from another
+====================================
+
+Starting in the :menuselection:`Inventory` module, select :menuselection:`Products --> Products` and
+then choose the product that will be replenished. Click the :guilabel:`Replenish` button on the top
+left of the product page and fill out the pop-up form as follows:
+
+- :guilabel:`Quantity`: the number of units that will be sent to the warehouse being replenished
+- :guilabel:`Scheduled Date`: the date that the replenishment is scheduled to take place
+- :guilabel:`Warehouse`: the warehouse that will be replenished
+- :guilabel:`Preferred Routes`: select `X: Supply Product from Y`, with *X* being the warehouse to
+  be replenished and *Y* being the warehouse that the product will be transferred from
+
+.. image:: warehouse_replenishment_transfer/product-replenishment-form.png
+   :align: center
+   :alt: The form for replenishing a product.
+
+Click :guilabel:`Confirm` and a delivery order will be created for the outgoing warehouse along with
+a receipt for the warehouse that will receive the product. Depending on the configuration settings
+for the outgoing and incoming warehouses, processing delivery orders and receipts will require
+between one and three steps. This document will detail how to process one-step deliveries and
+receipts.
+
+Process the delivery order
+--------------------------
+
+The first stage of a replenishment order is processing the delivery from the warehouse that the
+product is being transferred from. On the :menuselection:`Inventory` dashboard, select the
+:guilabel:`X to Process` button on the :guilabel:`Delivery Orders` card for the outgoing warehouse,
+then the delivery order created for the replenishment. On the delivery order page, click the
+:guilabel:`Check Availability` button in the top left to reserve the quantity of the product to be
+transferred. Once the delivery has been dispatched, click the :guilabel:`Validate` button to
+register the quantities shipped.
+
+.. image:: warehouse_replenishment_transfer/delivery-orders-card.png
```

Comment:
I'm going to leave this as-is just because the Inventory overview page can look different depending on a few factors: the number of warehouses created, the multi-step routes settings for those warehouses, etc. We're already telling the user what page to look for the card on and what the card will look like. If we show them a picture of the full UI, they could get confused because their "Delivery Orders" card appears in a different place than the one in the example.